src/ripple/overlay/impl/Manifest.h

```diff
@@ -0,0 +1,224 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_OVERLAY_MANIFEST_H_INCLUDED
+#define RIPPLE_OVERLAY_MANIFEST_H_INCLUDED
+
+#include <ripple/basics/BasicConfig.h>
+#include <ripple/basics/UnorderedContainers.h>
+#include <ripple/protocol/AnyPublicKey.h>
+#include <ripple/protocol/STExchange.h>
+#include <beast/utility/Journal.h>
+#include <boost/optional.hpp>
+#include <string>
+
+namespace ripple {
+
+/*
+    Validator key manifests
+    -----------------------
+
+    First, a rationale:  Suppose a system adminstrator leaves the company.
```

Comment:
We should reword the "_leaves the company_" bit since it implies that this feature is only relevant to companies, or worse still, Ripple Labs. Validator manifests are a great feature to anyone seeking to run a validator.